pom.xml

```diff
@@ -620,50 +684,19 @@
                 <goal>jar</goal>
               </goals>
             </execution>
-            <execution>
-              <id>default-test-jar</id>
```

Comment:
The main use we need to think about for the long term is the way that SDKs offer their own unit tests as test suites for runners. We may refactor but for now it makes sense for everything under `sdks` to offer the test jar.